package.json

```diff
@@ -164,6 +166,6 @@
     "test:lang": "gulp testlang",
     "update": "gulp update",
     "watch-streamer": "cd docs/static/streamer && tsc -t es6 --watch",
-    "prepare": "cd skillmap && npm install && cd .. && cd authcode && npm install && cd .. && cd multiplayer && npm install && cd .. && cd kiosk && npm install && cd .."
+    "prepare": "cd skillmap && npm install && cd .. && cd authcode && npm install && cd .. && cd multiplayer && npm install && cd .. && cd kiosk && npm install && cd ..  && cd teachertool && npm install && cd .."
```

Comment:
Not something for this PR, but I wonder if we can clean this up a bit somehow since there are a lot of repeated operations... Could we maybe make a script that this npm script points to? And it performs the directory change and install per whatever webapps we have configured?

Reply:
Agree. There's a lot of shared bits and pieces, plus some repetitive things like what we have here. Would be nice to go through and simplify the whole process of creating new apps at some point, but I'm not sure if that'll happen anytime soon :)